test_runner/performance/test_perf_pgbench.py

```diff
@@ -129,6 +176,7 @@ def test_pgbench_flamegraph(zenbenchmark, pg_bin, neon_env_builder, scale: int,
 # Run the pgbench tests against an existing Postgres cluster
 @pytest.mark.parametrize("scale", get_scales_matrix())
 @pytest.mark.parametrize("duration", get_durations_matrix())
+@pytest.mark.parametrize("filter", ["init", "simple-update", "select-only"])
```

Comment:
can I run `simple-update` without running `init` first?

Reply:
Currently, we use pre-created DB, so yes

Reply:
Where is it created? Does the scale parameter affect `simple-update`?

Reply:
> Where is it created?

Wherever we pass in connection string, this PR uses `BENCHMARK_CAPTEST_CONNSTR` and `BENCHMARK_RDS_CONNSTR`, so it's on captest env and on AWS RDS.

> Does the scale parameter affect simple-update?

Not directly, and if we run only `simple-update` test it doesn't affect it at all

Reply:
We use the same connstr for init and simple-update. So the database size is effectively "whatever was created by the last init run"?

Reply:
Yes (and if we run only `simple-update`, then the database will come from the previous run)

Reply:
I probably misunderstood your first question, and now I got lost 😄 

So

> can I run simple-update without running init first?

Yes, using `-k simple-update`, as we use in our workflow. And in this case, we won't know anything about real size

Reply:
Ok, I understand. Currently we run 3 steps:
1. `-k init`
2. `-k simple-update`
3. `-k select-only`

The duration+scale matrix for the `bench-captest` job is 1x1, so this is fine.

But the `bench` job runs all `remote_cluster` tests, in whatever order pytest decides on, and it's matrix is 2x1. Is it possible that It runs two inits with different sizes that override each other, and then two simple-updates that are intended to run with different sizes but actually run on whatever was left from the last init?

There's also some non-determinisim if simple-update and select-only are run in a different order with each other. Maybe select-only is slower if it runs on a database that has been simple-update'd?

FYI [this](https://pytest-dependency.readthedocs.io/en/stable/) might be helpful

Reply:
> But the bench job runs all remote_cluster tests, in whatever order pytest decides on, and it's matrix is 2x1. Is it possible that It runs two inits with different sizes that override each other, and then two simple-updates that are intended to run with different sizes but actually run on whatever was left from the last init?

You're right. `bench` job is a bit outdated for now (uses runner manually configured runner, for example). I will update it in the following PRs.

> FYI [this](https://pytest-dependency.readthedocs.io/en/stable/) might be helpful

Yep, I think I'll use either this or [that](https://pypi.org/project/pytest-order/)